servant-auth-server/src/Servant/Auth/Server/Internal/Cookie.hs

```diff
@@ -28,8 +29,9 @@ cookieAuthCheck ccfg jwtCfg = do
     cookies' <- lookup "Cookie" $ requestHeaders req
     let cookies = parseCookies cookies'
     xsrfCookie <- lookup (xsrfCookieName ccfg) cookies
-    xsrfHeader <- lookup (mk $ xsrfHeaderName ccfg) $ requestHeaders req
-    guard $ xsrfCookie `constTimeEq` xsrfHeader
+    when (((requestMethod req) /= methodGet) || not (xsrfExcludeGet ccfg)) $ do
+        xsrfHeader <- lookup (mk $ xsrfHeaderName ccfg) $ requestHeaders req
+        guard $ xsrfCookie `constTimeEq` xsrfHeader
```

Comment:
This file uses 2-spaces for indentation instead of 4 as is used here.

Reply:
Thanks, I am used to 4 spaces so it slipped in. Fixed.